src/types/tracker.d.ts

```diff
@@ -253,5 +253,70 @@ declare class Tracker {
     networkParameters?: NetworkParameters
   ): true | Error;
 
+  trackAssistantSubmit(
+    parameters: {
+      intent: string;
+      section?: string;
+    },
+    networkParameters?: NetworkParameters
+  ): true | Error;
+
+  trackAssistantLoadStarted(
+    parameters: {
+      intent: string;
+      section?: string;
+      intentResultId?: string;
+    },
+    networkParameters?: NetworkParameters
+  ): true | Error;
+
+  trackAssistantLoadFinished(
+    parameters: {
+      intent: string;
+      searchResultCount: number;
+      section?: string;
+      intentResultId?: string;
+    },
+    networkParameters?: NetworkParameters
+  ): true | Error;
+
+  trackAssistantResultClick(
+    parameters: {
+      intent: string;
+      searchResultId: string;
+      itemId?: string;
+      itemName?: string;
+      variationId?: string;
+      section?: string;
+      intentResultId?: string;
+    },
+    networkParameters?: NetworkParameters
+  ): true | Error;
+
+  trackAssistantResultView(
+    parameters: {
+      intent: string;
+      searchResultId: string;
+      numResultsViewed: number;
+      items?: ItemTracked[];
+      intentResultId?: string;
+      section?: string;
+    },
+    networkParameters?: NetworkParameters
+  ): true | Error;
+
+  trackAssistantSearchSubmit(
+    parameters: {
+      intent: string,
+      searchTerm: string;
+      userInput: string;
+      searchResultId: string;
+      groupId?: string;
```

Comment:
The slab docs mentions that this is inside filters object

Reply:
I kept this the same as trackSearchSubmit, I think it's fine since there is only one.